src/cli/MareTF.cpp

```diff
@@ -1032,6 +1039,48 @@ int main(int argc, const char* const argv[]) {
 				// Set inversion of green channel
 				options.invertGreenChannel = invertGreenChannel || invertGreenChannelAlt;
 
+				// Handle special case for HDRI --> cubemap conversion
+				if (convertFromHdri) {
+					::ElapsedTime hdriStopwatch;
+					options.flags |= vtfpp::VTF::Flags::FLAG_ENVMAP;
+					options.isCubeMap = true;
+
+					vtfpp::ImageFormat hdriFormat;
+					int hdriWidth, hdriHeight, hdriFrameCount;
+					std::vector<std::byte> hdriData = vtfpp::ImageConversion::convertFileToImageData(
+						sourcepp::fs::readFileBuffer(currentInputPath),
+						hdriFormat, hdriWidth, hdriHeight, hdriFrameCount);
+
+					int faceWidth = hdriHeight;
+					std::array<std::vector<std::byte>, 6> cubeFaces = vtfpp::ImageConversion::convertHDRIToCubeMap(
+						hdriData, hdriFormat, hdriWidth, hdriHeight, faceWidth);
+
+					vtfpp::VTF vtf = vtfpp::VTF::create(hdriFormat, hdriHeight, hdriHeight, options);
+					vtf.setFaceCount(true, true); // Spheremap is required pre-7.5, so pretend we have it in that case
+					
+					for (int face = 0; face < 6; face++) {
+						if (vtf.setImage(cubeFaces[face], hdriFormat, faceWidth, faceWidth, options.filter, 0, 0, face, 0)) continue;
+						tferr << "Failed to set face " << BOLD << face << END << tfendl;
+						return EXIT_FAILURE;
+					}
+
+					// Something should go here to insert the spheremap but whatever
+					// if (vtf.getFaceCount() == 7) {
+					// 	...
+					// }
+
+					vtf.computeMips(options.filter);
+
+					if (!vtf.bake(outputPath)) {
+						tferr << "Failed to TF input image at " << BOLD << currentInputPath << END << ". Is it a supported format?" << tfendl;
```

Comment:
don't need to ask about supported format because it's already been loaded earlier